### PR TITLE
KB-3640, KB-3780, KB-3784

### PR DESCRIPTION
**Pull request by NandiniAV**
KB-3640, KB-3780,  KB-3784

### CODE REVIEW

File: project/ws/app/src/lib/head/left-menu/left-menu.component.html [hunk 1]
```diff
@@ -14,28 +14,39 @@
     </a>
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/head/left-menu/left-menu.component.html:14]: please mention Jira ticket id's in PR title

File: project/ws/app/src/lib/head/left-menu/left-menu.component.scss [hunk 2]
```diff
@@ -91,6 +92,11 @@
 //   margin: unset !important;
 // }
 
+::ng-deep  .mat-expansion-panel:not([class*=mat-elevation-z]) {
+    background: rgba(0, 0, 0, 0.02) !important;
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/head/left-menu/left-menu.component.scss:96]: please add same on by creating a custom class as a parent

File: project/ws/app/src/lib/head/left-menu/left-menu.component.scss [hunk 3]
```diff
@@ -110,9 +116,10 @@
 }
 
 ::ng-deep .menuPannel .mat-expansion-panel-header {
-  padding: 8px 16px !important;
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/head/left-menu/left-menu.component.scss:113]: same as above

File: project/ws/app/src/lib/routes/home/components/filter/filter.component.ts [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+      this.competencyThemeList.map((titem: any) => {
+        if (titem && titem['selected']) {
+          titem['selected'] = false
+        }
+      })
+    }
+    if (this.competencySubThemeList) {
+      this.competencySubThemeList.map((sitem: any) => {
+        if (sitem && sitem['selected']) {
+          sitem['selected'] = false
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/filter/filter.component.ts:400]: please confirm, if you are using duplicate component then code should be revised as per new requirement

File: project/ws/app/src/lib/routes/home/components/search/search.component.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        </div>
+    </div>
+    <div class="filter-section flex">
+        <button mat-button type="button" (click)="sort()" class="apply-filter margin-right-s">
+            <mat-icon class="margin-right-xs">sort</mat-icon> Sort
+        </button>
+    </div>
+    <div class="filter-section flex">
+        <button mat-button type="button" (click)="openFilter()" class="apply-filter margin-right-s">
+            Apply Filter <mat-icon class="margin-right-xs">filter_list</mat-icon>
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/search/search.component.html:16]: same as above

File: project/ws/app/src/lib/routes/home/components/search/search.component.ts [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss'],
+})
+export class SearchComponent implements OnInit {
+  @Input() from: any = ''
+  @Output() handleApiData = new EventEmitter()
+  searchText = ''
+  filterVisibilityFlag = false
+  clearFilter = false
+  designationList: any[] = []
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/search/search.component.ts:19]: same as above

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html [hunk 7]
```diff
@@ -23,19 +37,26 @@
...
-        </div> -->
+        </div>
       </mat-expansion-panel-header>
 
       <div class="panel-body" *ngIf="user.editUser">
         <div class="inner-body">
           Primary Details <mat-icon class="ws-mat-black40-text">info_outline</mat-icon>
           <form [formGroup]="updateUserDataForm" class="margin-top-m">
-            <mat-grid-list cols="2" rowHeight="100px">
+            <mat-grid-list cols="2" rowHeight="60px">
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html:59]: make sure these changes should not impact existing functionality

File: project/ws/app/src/lib/head/left-menu/left-menu.component.html (hunk 1 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/head/left-menu/left-menu.component.html:14]: KB-3640, KB-3780, KB-3784 already mentioned

File: project/ws/app/src/lib/head/left-menu/left-menu.component.scss (hunk 2 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/head/left-menu/left-menu.component.scss:96]: Okay will do it

File: project/ws/app/src/lib/head/left-menu/left-menu.component.scss (hunk 3 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/head/left-menu/left-menu.component.scss:113]: Okay will do it

File: project/ws/app/src/lib/routes/home/components/search/search.component.html (hunk 5 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/components/search/search.component.html:16]: Filter is required here & have used which is required only

File: project/ws/app/src/lib/routes/home/components/search/search.component.ts (hunk 6 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/components/search/search.component.ts:19]: Ya I have cleaned up the code, just added as I do the integration further I will clean up the code

File: project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html (hunk 7 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/components/user-cards/user-card.component.html:59]: I have only added it corrected the extra space here

File: project/ws/app/src/lib/routes/home/components/filter/filter.component.ts (hunk 4 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/routes/home/components/filter/filter.component.ts:400]: Ya I have cleaned up the code, just added as I do the integration further I will clean up the code

File: project/ws/app/src/lib/head/left-menu/left-menu.component.scss (hunk 3 above)
Comment (NandiniAV) [reply project/ws/app/src/lib/head/left-menu/left-menu.component.scss:113]: menuPannel is the parent class have checked

**Pull request merged**